app/geant-exporter/ImportProcessConverter.cc

```diff
@@ -534,6 +551,97 @@ void ImportProcessConverter::add_table(const G4PhysicsTable* g4table,
     process_.tables.push_back(std::move(table));
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * While all other processes keep one lambda table per process,
+ * multiple scattering stores them by model. In Geant4, these tables are
+ * labeled as LambdaMod[i], where i is the model index.
+ *
+ * This method overwrites \c process_.tables to store a single lambda table
+ * that is a concatenation of the lambdaMod tables for Urban (low E) and
+ * WentzelVI (high E) msc models. This allows Celeritas to load the msc process
+ * as one lambda table, avoiding the need to write an exception for msc.
+ *
+ * \note
+ * At the interface---i.e. last bin of low energy lambda and first bin of high
+ * energy lambda---the cross-section values differ, despite their energy value
+ * being the same. To smooth out the combination, this bin is replaced by
+ * taking the average of both cross-section values.
+ */
+void ImportProcessConverter::merge_msc_tables()
+{
+    // Check that only 2 tables are stored (i.e. lambdas for low and high E)
+    // Check that low/high energy vectors are in ascending order of energy
+    // (The ordering is defined by PhysicsList)
+    CELER_ASSERT(process_.tables.size() == 2);
```

Comment:
Good for now, but better to handle arbitrary number of models inside the process (i.e., number of MSC models added via a physics list) for the purpose of generality.  It should also handle the case that we only construct a MSC process with only one (Geant4) model.